src/core/agent/services/connectionService.test.ts

```diff
@@ -478,8 +472,12 @@ describe("Connection service of agent", () => {
       .mockResolvedValue({ done: false });
     Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(true);
     jest.spyOn(Date.prototype, "getTime").mockReturnValueOnce(0);
+    const waitForCompletion = true;
     await expect(
-      connectionService.resolveOobi(`${oobiPrefix}${uuidToThrow}`)
+      connectionService.resolveOobi(
```

Comment:
small thing but can we rename this test to "should throw if oobi is not resolving and we explicitly wait for completion"